google-http-client/src/test/java/com/google/api/client/http/apache/ApacheHttpTransportTest.java

```diff
@@ -43,17 +47,16 @@ public void testApacheHttpTransport() {
     checkHttpClient(httpClient);
   }
 
-  public void testApacheHttpTransportWithParam() {
-    ApacheHttpTransport transport = new ApacheHttpTransport(new DefaultHttpClient());
-    checkHttpClient(transport.getHttpClient());
-  }
-
   public void testNewDefaultHttpClient() {
     checkDefaultHttpClient(ApacheHttpTransport.newDefaultHttpClient());
   }
 
   public void testRequestsWithContent() throws Exception {
-    ApacheHttpTransport transport = new ApacheHttpTransport(new MockHttpClient());
+    HttpClient mockClient = mock(HttpClient.class);
+    org.apache.http.HttpResponse mockResponse = mock(org.apache.http.HttpResponse.class);
```

Comment:
Why does `HttpResponse` need to be fully qualified?

Reply:
No need. :) Thanks.